apps/chat/src/constants/modal.ts

```diff
@@ -0,0 +1,32 @@
+import { UseDismissProps } from '@floating-ui/react';
+
+export const MOUSE_OUTSIDE_PRESS_EVENT: Pick<
```

Comment:
I'm not really sure. Do we need this constants?

Reply:
@Alexander-Kezik,  yes, we need to use in all our modals instead of new objects